backend/pyproject.toml

```diff
@@ -24,21 +24,21 @@ django-jinja = "2.11.0"
 django-lazy-managers = "0.0.1"
 django-lookup-property = "0.2.0"
 django-modeltranslation = "0.19.16"
-django-mptt = "0.17.0"
+django-mptt = "0.18.0"
 django-redis = "6.0.0"
-djangorestframework = "3.16.0"
-easy-thumbnails = "2.10"
+djangorestframework = "3.16.1"
+easy-thumbnails = "2.10.1"
 helsinki-profile-gdpr-api = "0.2.0"
 html2text = "2025.4.15"
 icalendar = "6.3.1"
 mjml-python = "1.3.5"
 psycopg2-binary = "2.9.10"
 python-dateutil = "2.9.0.post0"
-redis = { version = "5.3.0", extras = ["hiredis"] }
```

Comment:
Why was redis downgraded from `5.3.0` to `5.2.1`?

Reply:
[New version of celery is incompatible with newer redis versions](https://github.com/celery/celery/releases#:~:text=Limit%20redis%20to%20%3C%3Dv5.2.1%20to%20match%20Kombu%20by%20%40Nusnus%20in%20%239693).